.ci/wercker.sh

```diff
@@ -427,6 +427,17 @@ no-warning-imports-java-design-patterns)
   fi
   ;;
 
+no-error-pmd)
+  CS_POM_VERSION=$(mvn -e -q -Dexec.executable='echo' -Dexec.args='${project.version}' \
+                     --non-recursive org.codehaus.mojo:exec-maven-plugin:1.3.1:exec)
+  echo CS_version: ${CS_POM_VERSION}
+  checkout_from https://github.com/pmd/pmd.git
+  cd .ci-temp/pmd
+  mvn checkstyle:check
```

Comment:
as the last verification .....
please modify pmd code to cause violation and confirm that it is failing.

Reply:
Is `mvn -e compile checkstyle:check -Dcheckstyle.version=${CS_POM_VERSION}` good?  It produces this:
`[ERROR] Failed to execute goal org.apache.maven.plugins:maven-enforcer-plugin:3.0.0-M2:enforce (enforce-versions) on project pmd: Some Enforcer rules have failed.`

Reply:
we expect violations from maven checkstyle plugin.

Reply:
Should build ok: https://github.com/pmd/pmd/blob/b487ef94d09855dfb3bb34f79ba267d301980b6e/pom.xml#L96

Reply:
@nmancus1 , please share log (at gist.github.com) of checkstyle violation that result in build failure in pmd (please make any violation yourself).

Reply:
Just to be clear, you want me to clone pmd, make changes that will violate Checkstyle, run the script on my local pmd repo, and show a gist of Checkstyle violations?

Reply:
yes, show whole log of execution , I need to see that it fails.

Reply:
@romani https://gist.github.com/nmancus1/eb931a5bd761d04d97cb0c66a95a3add.js

Reply:
ok, looks good.
```
[INFO] --- maven-checkstyle-plugin:3.1.1:check (checkstyle-check)
 @ pmd-core ---
[INFO] There is 1 error reported by Checkstyle 8.31-SNAPSHOT 
with /net/sourceforge/pmd/pmd-checkstyle-config.xml ruleset.
[ERROR] src/main/java/net/sourceforge/pmd/lang/ast/Node.java:[1]
 (misc) NewlineAtEndOfFile: File does not end with a newline.
[INFO] ------------------------------------------------------------------------

```